### PR TITLE
Makefile log info

### PR DESCRIPTION
**Pull request by oharboe**
Part deux https://github.com/The-OpenROAD-Project/OpenROAD/pull/5876

Manual test locally:

```
$ make do-2_3_floorplan_tdms 
Running tdms_place.tcl, stage 2_3_floorplan_tdms
exec cp ./results/nangate45/gcd/base/2_2_floorplan_io.odb ./results/nangate45/gcd/base/2_3_floorplan_tdms.odb
Elapsed time: 0:00.13[h:]min:sec. CPU time: user 0.10 sys 0.02 (100%). Peak memory: 100444KB.
$ head -n 10 logs/nangate45/gcd/base/2_3_floorplan_tdms.log 
OpenROAD v2.0-16257-g81b545063 
Features included (+) or not (-): +Charts +GPU +GUI +Python
This program is licensed under the BSD-3 license. See the LICENSE file for details.
Components of this program may be licensed under more restrictive licenses which must be honored.
[INFO ORD-0030] Using 12 thread(s).
exec cp ./results/nangate45/gcd/base/2_2_floorplan_io.odb ./results/nangate45/gcd/base/2_3_floorplan_tdms.odb
Elapsed time: 0:00.13[h:]min:sec. CPU time: user 0.10 sys 0.02 (100%). Peak memory: 100444KB.
```

### CODE REVIEW

File: flow/scripts/util.tcl [hunk 1]
```diff
@@ -99,3 +99,19 @@ proc find_macros {} {
...
 }
+
+proc erase_non_stage_variables {stage_name} {
+  # "$::env(SCRIPTS_DIR)/stage_variables.py stage_name" returns list of
+  # variables to erase.
+  # 
+  # Tcl yaml package can't be imported in the sta/openroad environment:
+  # 
+  # https://github.com/The-OpenROAD-Project/OpenROAD/issues/5875
+  set variables [exec $::env(SCRIPTS_DIR)/stage_variables.py $stage_name]
```
Comment (maliberty) [inline flow/scripts/util.tcl]: This refers to stage_variables.py but I only see flow/scripts/variables.py in this PR.

Comment (oharboe) [reply flow/scripts/util.tcl]: oops, force pushed fix to remove it.

**Comment by maliberty** [comment]
You appear to have just wrecked this PR (all the content gone).  Once you have requested a review please don't force push.  It destroys the ability to see changes since the last review.

**Comment by oharboe** [comment]
> You appear to have just wrecked this PR (all the content gone). Once you have requested a review please don't force push. It destroys the ability to see changes since the last review.

Normally I'm loath to force push, but in this case I had a giant commit that shouldn't be there.

From there it was a matter of picking a poison: a giant revert commit or a force push. I chose the latter.

**Comment by maliberty** [comment]
The content is entirely different and doesn't seem to match the title.  Is this right?

**Comment by oharboe** [comment]
> The content is entirely different and doesn't seem to match the title. Is this right?

Yes, this PR is now a single concern where the title is congruent with the single(tiny) commit.

"Makefile log info": This matches the commit: tweak logging info so that the number of threads specified in "-threads" is logged into .log.

**Review by maliberty — APPROVED** [review]

**Pull request merged**